docs/release-notes.md

```diff
@@ -26,6 +26,7 @@
 * [1224912] (https://bugzilla.redhat.com/show_bug.cgi?id=1224912) - Filter "Last modified by translators other than <user>"
 * [1213630](https://bugzilla.redhat.com/show_bug.cgi?id=1213630) - Webhook header needs to include cryptographic signature in header for identification
 * [1214502](https://bugzilla.redhat.com/show_bug.cgi?id=1214502) -  RFE: Grant project creation permission to certain sets of users
+* [ZNTA-108](https://zanata.atlassian.net/browse/ZNTA-108) - Glossary management
```

Comment:
The title of https://zanata.atlassian.net/browse/ZNTA-108 ought to be
improved too.

On 6 October 2015 at 09:57, David Mason <notifications@github.com> wrote:

> In docs/release-notes.md
> <https://github.com/zanata/zanata-server/pull/986#discussion_r41212899>:
>
> > @@ -26,6 +26,7 @@
> >  * [1224912] (https://bugzilla.redhat.com/show_bug.cgi?id=1224912) - Filter "Last modified by translators other than <user>"
> >  * [1213630](https://bugzilla.redhat.com/show_bug.cgi?id=1213630) - Webhook header needs to include cryptographic signature in header for identification
> >  * [1214502](https://bugzilla.redhat.com/show_bug.cgi?id=1214502) -  RFE: Grant project creation permission to certain sets of users
> > +* [ZNTA-108](https://zanata.atlassian.net/browse/ZNTA-108) - Glossary management
>
> All this work deserves more than 2 words :)
>
> Maybe Improved glossary management: add, edit and delete individual
> glossary entries
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/986/files#r41212899>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat